demos/About Window/main.vala

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
```

Comment:
The added whitespace is incorrect I think

Reply:
It works as well, not sure it is a good style though.